src/dotnet/APIView/APIViewWeb/LeanControllers/APIRevisionsTokenAuthController.cs

```diff
@@ -134,4 +134,24 @@ public async Task<ActionResult<string>> GetAPIRevisionTextAsync(
             return StatusCode(StatusCodes.Status500InternalServerError, "Failed to generate review text");
         }
     }
+
+    [HttpGet("{revisionId}/getReviewId", Name = "GetReviewId")]
+    public async Task<ActionResult> GetReviewIdForRevision(string revisionId)
```

Comment:
What's the use case of this REST API?

Reply:
sorry Praveen I changed all this code, so resolving this conversation :)